### PR TITLE
Helper fixes for external projects.

### PR DESCRIPTION
**Pull request by oliverchang**
- Add ":v1" suffix to Dockerfile template.
- Support relative paths for external project directories.

### CODE REVIEW

File: infra/templates.py [hunk 1]
```diff
@@ -47,7 +47,7 @@
 """
 
 EXTERNAL_DOCKER_TEMPLATE = """\
-FROM gcr.io/oss-fuzz-base/%(base_builder)s
+FROM gcr.io/oss-fuzz-base/%(base_builder)s:v1
```
Comment (oliverchang) [inline infra/templates.py:50]: This applies to OSS-Fuzz projects as well, but I think that's a good thing to be more explicit going forward. It's not something that changes very often either.

File: infra/helper.py [hunk 2]
```diff
@@ -135,15 +135,17 @@ def corpus(self):
 def main():  # pylint: disable=too-many-branches,too-many-return-statements
   """Gets subcommand from program arguments and does it. Returns 0 on success 1
   on error."""
-  os.chdir(OSS_FUZZ_DIR)
```
Comment (jonathanmetzman) [inline infra/helper.py:138]: Why don't we need this anymore?

**Review by jonathanmetzman — APPROVED** [review]
LGTM

File: infra/helper.py (hunk 2 above)
Comment (oliverchang) [reply infra/helper.py:138]: This is still here, just moved to line 145.

Comment (jonathanmetzman) [reply infra/helper.py:138]: ACK

**Review by CryptoGodfatherVA2 — APPROVED** [review]

**Pull request merged**